src/epics/wiki.js

```diff
@@ -16,35 +25,46 @@ const fetchAllWikis = ( action$ ) => (
 				.map( ( ajaxResponse ) => {
 					const wikis = Object.keys( ajaxResponse.response.sitematrix )
 						.filter( ( key ) => {
-							return ( !isNaN( parseFloat( key ) ) && isFinite( key ) ) || key === 'specials';
+							return ( !isNaN( parseFloat( key ) ) && isFinite( key ) );
 						} )
 						.map( ( key ) => {
-							// Special is different for some reason.
-							if ( key === 'specials' ) {
-								return {
-									site: ajaxResponse.response.sitematrix[ key ]
-								};
-							}
-
 							return ajaxResponse.response.sitematrix[ key ];
 						} )
 						.reduce( ( state, data ) => (
 							[
 								...state,
 								...data.site.map( ( item ) => (
-									new Wiki( {
+									{
 										id: item.dbname,
-										domain: new URL( item.url ).hostname
-									} )
+										domain: new URL( item.url ).hostname,
+										type: item.code,
```

Comment:
I didn't know what to call `code` in this two instances. Any suggestions?

Reply:
This is wikitionary vs wikipedia, etc.? If so I would call `wikiFamily`

Reply:
or just `family`

Reply:
sounds good to me, `family` works